File: ASSIGNMENTS/Core java/MyOwnAutoShopProject/src/com/sonata/AutoShop/Car.java
package com.sonata.AutoShop;

public class Car {
	int speed;
	double regularPrice;
    String color; 

Car(){}
Car(int sp,double rprice,String col){
   this.speed=sp;
   this.regularPrice=rprice;
   this.color=col;
   
}
public void getSalePrice()
{
   System.out.println(regularPrice);
}
public void display()
{
   
   System.out.println(speed);
   System.out.println(regularPrice);
   System.out.println(color);
}

}

File: ASSIGNMENTS/TypecriptAssignments/Car.ts
 export class Car{
	speed:number;
	regularPrice:number;
	color:string;
	constructor(s:number,rp:number,c:string){
		this.speed=s;
		this.regularPrice=rp;
		this.color=c;
	}
	getSalePrice():number{
        	return 0; 
	}
	display():void{
		console.log("initial speed:"+this.speed);
		console.log("color:"+this.color);
		console.log("initial price:"+this.regularPrice);
	}
}